x-pack/plugins/reporting/server/core.ts

```diff
@@ -127,6 +127,7 @@ export class ReportingCore {
 
     this.getContract = () => ({
       usesUiCapabilities: () => config.roles.enabled === false,
+      registerExportTypes: (id: any) => id,
```

Comment:
regular type inference works here:
```suggestion
      registerExportTypes: (id) => id,
```